.github/workflows/stale-tasks.yml

```diff
@@ -0,0 +1,26 @@
+# This is a workflow that runs periodically to mark issues/prs with no recent activity as stale, it also closes/removes label based on following activity
+
+name: Mark and close stale issues/PRs
+on:
+  schedule:
+  - cron: '0 0 * * *' # runs daily at 12:00 am UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-label: stale
+          stale-pr-label: stale
+          days-before-stale: 45
+          days-before-close: 45
```

Comment:
Is `days-before-close` started once stale label is added?

Reply:
yes ... description found in the docs: https://github.com/actions/stale

    days-before-close:  Idle number of days before closing stale issues/PRs